pynbody/sph/__init__.py

```diff
@@ -181,7 +181,7 @@ def rho(self):
 
 class Kernel(object):
 
-    def __init__(self):
+    def __init__(self,type='spline'):
```

Comment:
This option should not be present, rather a subclass should be implemented for a different kernel type